src/routes/RequestsRoute.js

```diff
@@ -173,6 +177,12 @@ class RequestsRoute extends React.Component {
       path: 'cancellation-reason-storage/cancellation-reasons',
       records: 'cancellationReasons',
     },
+    staffSlips: {
```

Comment:
Is this a new backend interface dependency, it appears to be a new endpoint for this module, and there doesn’t appear to be an entry in the package.json for it?

Reply:
I agree, @marcjohnson-kint ; it looks like we need to add a dependency on mod-circulation-storage.